src/app/components/elements/Assignments.tsx

```diff
@@ -68,7 +68,7 @@ export const AssignmentCard = ({assignment}: {assignment: AssignmentDTO}) => {
                     <strong>{topics.length === 1 ? "Topic" : "Topics"}:</strong>{" "}
                     {topics.join(", ")}
                 </p>}
-                {boardStagesAndDifficulties.length > 0 && <p className="mb-0">
+                {boardStagesAndDifficulties.length > 0 && <div className="mb-0">
```

Comment:
Is this change really needed?

Reply:
With it as <p>, we get "Warning: validateDOMNesting(...): <table> cannot appear as a descendant of <p>", so I changed to a <div>

Reply:
Cool, as long as you've checked that they both look the same, this is fine :-)